haas/ext/switches/nexus.py

```diff
@@ -203,6 +205,30 @@ def get_port_networks(self, ports):
             result[k] = networks
         return result
 
+    def _save_running_config(self):
+        """saves the running config to startup config"""
+
+        self.console.sendline('copy running-config startup-config')
+        self.console.expect('Copy complete')
+        logger.debug('Copy succeeded')
+
+    def _get_config(self, config_type):
+        """returns the requested configuration file from the switch"""
+
+        self.console.sendline('terminal length 0')
+        self.console.expect(r'[\r\n]+.+# ')
+        self.console.sendline('show ' + config_type + '-config')
+        self.console.expect(r'[\r\n]+.+# ')
+        config = self.console.after
+        lines_to_remove = 9
+        # the startup config file always has a couple of extra lines that
+        # we need to remove.
```

Comment:
Would it be possible to exclude these based on text rather than always getting rid of a fixed number of lines?